adele-booking/src/main/java/com/microservicesteam/adele/booking/domain/BookingService.java

```diff
@@ -0,0 +1,94 @@
+package com.microservicesteam.adele.booking.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.microservicesteam.adele.booking.boundary.web.EventPublisher;
+import com.microservicesteam.adele.messaging.EventBasedService;
+import com.microservicesteam.adele.ticketmaster.commands.BookTickets;
+import com.microservicesteam.adele.ticketmaster.events.TicketsBooked;
+import com.microservicesteam.adele.ticketmaster.events.TicketsCancelled;
+import com.microservicesteam.adele.ticketmaster.events.TicketsCreated;
+import com.microservicesteam.adele.ticketmaster.model.BookedTicket;
+import com.microservicesteam.adele.ticketmaster.model.FreeTicket;
+import com.microservicesteam.adele.ticketmaster.model.Position;
+import com.microservicesteam.adele.ticketmaster.model.Ticket;
+
+@Service
+public class BookingService extends EventBasedService {
+
+    private final BookingIdGenerator bookingIdGenerator;
+    private final EventPublisher eventPublisher;
+    Map<Position, Ticket> ticketRepository;
```

Comment:
As I can see this is not `private final`, because it's checked directly in the corresponding unit tests. At least it can be `final` or maybe you can add a package level accessor method for this map similarto `getTicketsStatus`?